libcudacxx/include/cuda/std/__cccl/ptx_isa.h

```diff
@@ -34,29 +34,59 @@
 // PTX ISA 8.7 is available from CUDA 12.8
 // The first define is for future major versions of CUDACC.
 // We make sure that these get the highest known PTX ISA version.
+// For clang cuda check https://github.com/llvm/llvm-project/blob/release/XX.x/clang/lib/Driver/ToolChains/Cuda.cpp
+// getNVPTXTargetFeatures
 #if _CCCL_CUDACC_AT_LEAST(13, 0)
 #  define __cccl_ptx_isa 870ULL
 // PTX ISA 8.7 is available from CUDA 12.8, driver r570
-#elif _CCCL_CUDACC_AT_LEAST(12, 8)
+#elif _CCCL_CUDACC_AT_LEAST(12, 8) && !_CCCL_CUDA_COMPILER(CLANG, <, 20)
 #  define __cccl_ptx_isa 870ULL
 // PTX ISA 8.5 is available from CUDA 12.5, driver r555
-#elif _CCCL_CUDACC_AT_LEAST(12, 5)
+#elif _CCCL_CUDACC_AT_LEAST(12, 5) && !_CCCL_CUDA_COMPILER(CLANG, <, 19)
 #  define __cccl_ptx_isa 850ULL
 // PTX ISA 8.4 is available from CUDA 12.4, driver r550
-#elif _CCCL_CUDACC_AT_LEAST(12, 4)
+#elif _CCCL_CUDACC_AT_LEAST(12, 4) && !_CCCL_CUDA_COMPILER(CLANG, <, 19)
 #  define __cccl_ptx_isa 840ULL
 // PTX ISA 8.3 is available from CUDA 12.3, driver r545
-#elif _CCCL_CUDACC_AT_LEAST(12, 3)
+#elif _CCCL_CUDACC_AT_LEAST(12, 3) && !_CCCL_CUDA_COMPILER(CLANG, <, 18)
 #  define __cccl_ptx_isa 830ULL
 // PTX ISA 8.2 is available from CUDA 12.2, driver r535
-#elif _CCCL_CUDACC_AT_LEAST(12, 2)
+#elif _CCCL_CUDACC_AT_LEAST(12, 2) && !_CCCL_CUDA_COMPILER(CLANG, <, 18)
 #  define __cccl_ptx_isa 820ULL
 // PTX ISA 8.1 is available from CUDA 12.1, driver r530
-#elif _CCCL_CUDACC_AT_LEAST(12, 1)
+#elif _CCCL_CUDACC_AT_LEAST(12, 1) && !_CCCL_CUDA_COMPILER(CLANG, <, 17)
 #  define __cccl_ptx_isa 810ULL
 // PTX ISA 8.0 is available from CUDA 12.0, driver r525
-#elif _CCCL_CUDACC_AT_LEAST(12, 0)
+#elif _CCCL_CUDACC_AT_LEAST(12, 0) && !_CCCL_CUDA_COMPILER(CLANG, <, 17)
 #  define __cccl_ptx_isa 800ULL
+// PTX ISA 7.8 is available from CUDA 11.8, driver r520
```

Comment:
Question: Why are the lines below added and not modified? Did we remove those lines in preparation for the next release?

Reply:
Those lines were removed when dropping support for CTK pre 12.0 but I found it sensible to have these in here because we do support down to clang-14 at least and it does not cost anything

Reply:
We removed those lines with the nvcc 11.x support here: https://github.com/NVIDIA/cccl/pull/3596/files#diff-c1970f9174ca95aa30e79850d1fe200344dc0f26e6702c64564aca2e72f4ec0f

I think it's fine to re-add those lines.